spec/spec_helper.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-$VERBOSE = nil # silence loud Ruby 2.7 deprecations
+$VERBOSE = nil unless ENV['RUBY_LOUD'] # silence loud Ruby 2.7 deprecations
```

Comment:
![loud](https://vignette.wikia.nocookie.net/dilbert/images/5/56/Loud_Howard.jpg/revision/latest/window-crop/width/200/x-offset/0/y-offset/0/window-width/400/window-height/400?cb=20110312172247)

this is great

Reply:
i guess Ruby 2.7.2 changes this behavior though. we can't win on this one.